File: 1). DSA/1). Data Structures/01). Array/C++/_001)_Search_Element.cpp
#include <iostream>
using namespace std;

int search(int arr[], int n, int x)
{
	for(int i = 0; i < n; i++)
	{
		if(arr[i] == x)
			return i;
	}

	return -1;
}

int main()
{
	int arr[] = {20, 5, 7, 25}, x = 5;
	cout<<"Searched Index = "<<search(arr, 4, x)<<endl;
}

File: 1]. DSA + CP/1]. DSA/1]. Data Structures/01]. Array/Dart/_001)_Search_Element.dart
int searchElement(var arr, var element) {
  for (var i = 0; i < arr.length; i++) {
    if (arr[i] == element) {
      return i;
    }
  }
  return -1;
}

main() {
  var arr = [1, 20, 33, 47, 5, 67, 71, 85, 9, 10];
  var result = searchElement(arr, 71);
  if (result == -1) {
    print('Element not found!');
  } else {
    print('Element found at index $result.');
  }
}
